extensions/answer_summarizers/models.py

```diff
@@ -239,3 +291,42 @@ def calculate_from_state_answers_dict(self, state_answers_dict):
             state_answers_dict['interaction_id'],
             self.id,
             categorized_answer_frequency_lists)
+
+
+class TopNUnresolvedAnswersByFrequency(BaseCalculation):
+    """Calculation for the top unresolved answers by frequency
+    The output from this calculation is a ranked list of unresolved answers, by
```

Comment:
by frequency --> in descending order of frequency

Reply:
Done.